i18n/shuffle/fwk/service/movepreferences/MovePreferencesService.properties

```diff
@@ -9,3 +9,13 @@ error.heightfeeders=Bad height for feeders.
 text.threshold=Threshold
 text.autoswappaint=Switch to Paint Mode Automatically
 text.mobilemode=Toggle Mobile Mode
+tooltip.simulations=Select the number of drop simulations to be done for every move (do not put too high!) 
+tooltip.autocompute=Check to update the best move whenever something changes
+tooltip.autoswitch=Check to automatically switch to Paint Mode after doing a move
+tooltip.mobilemode=Check if you are using the mobile version of the game
+tooltip.effects=Select an effect
+tooltip.effectcheck=Check to use the threshold for the selected effect
```

Comment:
Actually, the checkbox here is unrelated to the threshold. If this checkbox is unchecked for a given effect it will never even be allowed to try to occur - that is, its effective threshold is forced to be infinite.